executor/simple.go

```diff
@@ -641,8 +641,22 @@ func (e *SimpleExec) executeReleaseSavepoint(s *ast.ReleaseSavepointStmt) error
 	return nil
 }
 
+func (e *SimpleExec) setCurrentUser(users []*auth.UserIdentity) {
+	sessionVars := e.ctx.GetSessionVars()
+	for i, user := range users {
```

Comment:
Do you want to find the current user and set it with current session?
Why not break if find it?

Reply:
In order to prevent an error when calling current_user multiple times, such as "revoke 'r1' from current_user(),current_user() ;" . Test cases for this scenario have been added.